packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -797,6 +797,7 @@ class StripeHelper {
    * Format the plan name for display
    *
    * @param {Plan} plan
+   * @returns {Promise<string>}
```

Comment:
Oops, I wasn't very specific on which function needed a more thorough return type for the type-checker to have caught it. It's the `subscriptionsToResponse` return type that is too vague with ` @returns {Promise<object[]>}`. That allows any arbitrary list of objects which can include promises. It probably would be a pain to add the actual type though.

Reply:
Ah, I see. It probably would be a bit of a pain to set up that return object